packages/wrangler/src/api/form_data.ts

```diff
@@ -105,6 +106,10 @@ export function toFormData(worker: CfWorkerInit): FormData {
     });
   });
 
+  if (bindings.raw) {
+    metadataBindings.push(bindings.raw);
```

Comment:
This will push an array, is that expected?

Reply:
Ah, no, forgot I need to do `...bindings.raw`